actors/builtin/miner/miner_state.go

```diff
@@ -1061,7 +1061,7 @@ func quantizeUp(e abi.ChainEpoch, unit abi.ChainEpoch) abi.ChainEpoch {
 
 // Rounds e to the nearest exact multiple of the quantization unit, rounding down.
 // Precondition: unit >= 0 else behaviour is undefined
-func quantizeDown(e abi.ChainEpoch, unit abi.ChainEpoch) abi.ChainEpoch {
+func quantizeDown(e abi.ChainEpoch, unit abi.ChainEpoch) abi.ChainEpoch { //nolint:deadcode,unused
```

Comment:
I've kept simple unused methods around with nolint directives.  Happy to delete if you think its a good idea.

Reply:
You can delete this one.